app/models/like.rb

```diff
@@ -1,2 +1,12 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
```

Comment:
- [ ] Optional you do not need a class_name 'Post'.
 [ ]  It will be nice to have Rails active callbacks added